Tests/AMPStorageTests.m

```diff
@@ -0,0 +1,144 @@
+//
+//  AMPStorageTests.m
+//
+//
+//  Created by Dante Tam on 7/22/21.
+//  Copyright © 2021 Amplitude. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import "AMPStorage.h"
+
+@interface AMPStorageTests : XCTestCase
+    
+@end
+
+@implementation AMPStorageTests {}
+
+NSString *exampleEvent = @"{\n    \"api_properties\" :     {\n        \"ios_idfv\" : \"2D623DDA-4AC8-4A1E-BB4B-8264CB2AB31A\"\n    },\n    \"carrier\" : \"Unknown\",\n    \"country\" : \"United States\",\n    \"device_id\" : \"8C027883-5AA1-43F8-8FB8-7837EBE519C4\",\n    \"event_type\" : \"testEvent\",\n \"user_id\" : \"userId\",\n    \"user_properties\" :     {\n    },\n    \"uuid\" : \"41EEDEBB-3A7E-4EDF-BEFC-689D7912BCF3\"\n}";
+
+- (void)setUp {
+    [super setUp];
+}
+
+- (void)tearDown {
+    [super tearDown];
+    NSString* dir = [AMPStorage getAppStorageAmpDir:@""];
+    NSFileManager *fileMgr = [NSFileManager defaultManager];
+    NSArray *fileArray = [fileMgr contentsOfDirectoryAtPath:dir error:nil];
+    for (NSString *filename in fileArray)  {
+        [fileMgr removeItemAtPath:[dir stringByAppendingPathComponent:filename] error:NULL];
```

Comment:
Does this only remove the file in the default instance folder, do you think we need to do anything for other instances we created in the test?

Reply:
This should remove every file and directory under `.../Application Support/com.apple.dt.xctest.tool/`

Reply:
What else should we do here @yuhao900914 ? Let me know if I'm missing something.

Reply:
Oh my bad, yeah, this should works fine.